tests/CMakeLists.txt

```diff
@@ -378,6 +378,11 @@ add_legacy_tests(
   bug_2517
 )
 
+add_legacy_tests(
+  FOLDER tests/unit NAMES
+  igraph_rich_club
```

Comment:
There's another command that adds tests from the `tests/unit` folder, starting from line 282. `igraph_rich_club` should be added there instead of starting a new section here.